infrastructure/main.tf

```diff
@@ -24,3 +24,9 @@ terraform {
   }
 }
 
+resource "aws_s3_bucket" "lambda_area_bucket" {
+  bucket = "org-hca-logs-lambda-deployment"
```

Comment:
Given that we are going to be deploying this into two different AWS accounts, this will need to be namespaced by deployment environment (unless you want a setup where you have to grant both accounts access to this bucket).